int64 id
ltlplanner/automaton automata
